aptos-move/move-examples/mint_nft/step1/sources/nft_step1.move

```diff
@@ -0,0 +1,144 @@
+/// This module is the step 1 of our NFT Move tutorial. In this module, we go over how to create a collection, a token,
```

Comment:
I wonder if we can move this to developer-docs-site (specifically mint nft tutorial docs). Having instructions for how to use modules is useful but can become outdated if we update docs (or vice versa)

Reply:
> I wonder if we can move this to developer-docs-site (specifically mint nft tutorial docs). Having instructions for how to use modules is useful but can become outdated if we update docs (or vice versa)

Yes, please do!